static/schemas/2.0/relations.2.0.json

```diff
@@ -0,0 +1,23 @@
+{
+    "title": "Relations.2.0",
+    "definitions": {
+        "Relation": {
+            "description": "A key value set of relations",
+            "type": "object",
+            "properties": {
+                "contains": {
```

Comment:
The scope of this PR is just manual rack management and I don't know if we want to also allow access to the other relation types at this point or if that's in the scope of this PR